src/plugins/janus_videoroom.c

```diff
@@ -11098,7 +11093,7 @@ static void *janus_videoroom_handler(void *data) {
 								json_integer_value(sc_substream) < 0 || json_integer_value(sc_substream) > 2) {
 							JANUS_LOG(LOG_ERR, "Invalid element (substream/spatial_layer should be 0, 1 or 2)\n");
 							error_code = JANUS_VIDEOROOM_ERROR_INVALID_ELEMENT;
-							g_snprintf(error_cause, 512, "Invalid value (substream/spatial_layer should be 0, 1 or 2)");
+							g_snprintf(error_cause, 512, "Invalid value (substream/spatial_layer should be 0, 1 or 2)");							
```

Comment:
There's a weird series of tabs/spaces at the end of this line?

Reply:
I suspect it's an issue with github, can not find it in code

Reply:
If it shows, it's there, maybe it's your IDE that isn't displaying it correctly?

Reply:
Right, my bad. Exactly the same line appears 3 times. Fixed